Makefile

```diff
@@ -9,8 +9,7 @@ package: \
 	openedge-agent/package.zip \
 	openedge-remote-mqtt/package.zip \
 	openedge-function-manager/package.zip \
-	openedge-function-python27/package.zip \
-	openedge-function-python36/package.zip \
+	openedge-function-python/package27.zip \
```

Comment:
python36.zip ?

Reply:
This is a little tricky which means package27.zip just use for detecting whether package27.zip and package36.zip exist based on the assumption package27.zip and package36.zip all exist in the same time.